test/example-types/results.js

```diff
@@ -60,6 +60,7 @@ describe('results', () => {
         from: 0,
         size: 10,
         explain: false,
+        track_total_hits: true,
```

Comment:
Fixing this made it clear that we should seriously reconsider the whole `sequentialResultsTest` thing 😅